src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
```

Comment:
calculate rental cost
```suggestion
  let rentalCost = 0;
```